pkg/piperutils/fileUtils_test.go

```diff
@@ -96,3 +96,55 @@ func runInTempDir(t *testing.T, nameOfRun, tempDirPattern string, run func(t *te
 
 	t.Run(nameOfRun, run)
 }
+
+func TestExcludeFiles(t *testing.T) {
+	t.Parallel()
+	t.Run("nil slices", func(t *testing.T) {
+		t.Parallel()
```

Comment:
Why is another `t.Parallel()` needed here?

Reply:
Something I learned from Fabian. See also https://github.com/SAP/jenkins-library/blob/master/DEVELOPMENT.md#test-parallelization